packages/common/core-interfaces/src/test/types/jsonable.ts

```diff
@@ -0,0 +1,173 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Jsonable } from "../..";
+
+declare function foo<T>(jsonable: Jsonable<T>): void;
```

Comment:
is there a corresponding delete for this change?

Reply:
oh. it looks like you didn't remove the old type yet.

Reply:
since this is next you can also remove too. just make sure you add a breaking note as well